src/pages/main/components/Carousel.tsx

```diff
@@ -52,9 +50,9 @@ export default Carousel;
 const CarouselContentWithSummaryWrapper = styled.div``;
 
 const CarouselWithButtonLayout = styled.section`
-  margin-left: 21.8rem;
   width: 100%;
   margin-right: 21.8rem;
+  margin-left: 21.8rem;
```

Comment:
[p5] 이거 줄여서
```suggestion
margin: 0 21.8rem;
```
으로 쓸 수 있어용